sycl/test/abi/layout_accessors_host.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -fsycl -c -fno-color-diagnostics -Xclang -fdump-record-layouts %s | FileCheck %s
+// RUN: %clangxx -fsycl -c -fno-color-diagnostics -Xclang -fdump-record-layouts %s -o %t.out | FileCheck %s
```

Comment:
Note for reviewers: most of `abi` tests started to fail with `-fsyntax-only` because output from `clang` was different. I haven't investigated it and simply decided to switch to `-o %t.out` instead.